### ISSUE TITLE
Multi-Attack vs Multi-Target

### ISSUE THREAD

**Issue by Groumy** [openlegend/core-rules#47]
It's not clear what the difference is between the two.

In the combat section, multi-attack refer as picking multiple targets, but the multi-attack feat refer as making multiple attacks.

And then there's the Multi-target attack feat that reduce the penalty for including multiple targets (like what the multi-attack in the combat section refer to).

So to make it clear, if I target multiple targets in a Melee Attack doest that mean we have to roll once and compare to the defense of every targets ? As if I made a cleave attack that just hit other creatures . And if I use the Multi-attack feat grand me _separate_ attack rolls on every target ?

If that's it, could one the attacks of the multi-attack feat target multiple targets ?? Could one of the attack be a bane one ? 

Tl;dr; Is the multi-attack feat simply a way to perform multiple actions in your round ? Or simply a duplicate of multi-target with worse inconvenient ?

**Comment by DMsShadow** [comment]
My understanding of this has always been that both feats and the rules section refer to multiple attacks (which can be either the same or multiple targets) and the disadvantage etc. works out the same. But perhaps it does need a bit of standardization on the entries so as not to cause confusion.

**Comment by brianfeister** [comment]
@DMsShadow - the two are intentionally different. Being able to "focus fire" and attack one target multiple times has more of an impact on combat, so `Multi-Attack` has a higher cost.

That's the intent anyway...

**Comment by istabosz** [comment]
@Groumy This example was recently added to the multi-attack specialist feat to clarify a few things:

> Vax the Deathbringer has Multi Attack Specialist (Tier VI). He declares that he will be making three attacks this round. Therefore, all of his attacks suffer disadvantage 3 (3 x 3 = 9, minus 6 for feat tier 6). His first attack is a necromantic burst targeting a 10'-square. Because of the area of effect, Vax suffers an additional disadvantage 2, making his total disadvantage for that attack 5. For his second attack, Vax casts a spell of blindess on a single foe, making an action roll at disadvantage 3. For his final attack, Vax moves in to melee and uses his touch of death on three foes. Targeting 3 foes incurs an additional disadvantage 3, making his final attack roll suffer a total of disadvantage 6.

So, yes, multi-attack allows multiple attack actions. Some could be damaging, some bane attacks. Some melee, some ranged. Some might use multi-targeting, some might not. Or, as @brianfeister mentioned, you could just use the feat to make a bunch of attacks against the same target.

**Comment by Groumy** [comment]
Ok that help to better understand the difference.

Is there a reason to limit to attacks ? I mean, if I can attack with different abilities on the same turn, some "mundane" and some "supernatural". It would make sense to be able to invoke a boon too, no ?!

I ask, because if the feat was Multi-Actions, rather then Multi-Attack the distinction with Multi-Target Specialist would be easier.

It could also be used to mimic a speedster making multiple actions in a blink of an eye. 

Not sure what repercussion this would cause though

**Comment by DMsShadow** [comment]
Just for my own clarity, are we saying that a **Multi-Target** attack is essentially one attack that hits multiple foes, either because they are all standing around you (melee), you have notched multiple arrows/multiple bolts of energy in a close area (ranged/supernatural), or it has an area if effect (supernatural). So you'd do one attack roll and see who it hits.

A **Multi-Attack** on the other hand is intended for multiple, discrete attacks that could be on different or the same target, and could even use different attributes, banes and be area effect spells etc. Hence the higher cost to use them.

Have I got that right?

**Comment by brianfeister** [comment]
You are correct, sir.

**Comment by LordRyst** [comment]
I think the main point of confusion is that Multi-Attack is not in the main combat section, but doesn't seem to require the feat to enable, just to reduce penalties. This isn't exactly stated, as in the effect section of the feat it says you can do the multi attack. Can we move the enabling section to combat and only give the cost reduction in the feat?

**Comment by brianfeister** [comment]
@LordRyst I'm reluctant to do that just because it's so powerful and I kind of like the idea that if you're not doing it, you're not actually "missing out", you just haven't specialized that way.

**Comment by Groumy** [comment]
I still think that Multi-Attack sound become Multi-Action.

This would solve the confusion between Multi-Attack and Multi-Target, since Multi-Action clearly refer to different actions and target refer to a single action.

I don't think there's a balance problem in invoking boon as place of one of the attacks that Multi-Attack already grant.

Lastly, if multiple-actions became a standard rule and the feat simply a way to limit the penalty, we could add that performing multiple-actions is a focus action or something to add an other penalty.

And to conclude and try to convince all of you that I'm not crazy and this is a good idea, I'd like to bring in the fact that Savage Worlds do just that. You can take as many actions as you want, but for each action beyond the first to suffer a cumulative -2 modifier on all actions. So you see I'm not crazy ;)

**Comment by brianfeister** [comment]
Yeah and I'm all for generalized terms @Groumy, but this is one where I'm just not feeling it. I want to be able to refer dual-wielders and multi-attack flurry people here and it just feels weird to call it "Multi-Action".  I'll re-open if I have a change of heart but closing for now
